### PR TITLE
Added information about Powershell AU module.

### PR DESCRIPTION
**Pull request by majkinetor**

### CODE REVIEW

**Comment by majkinetor** [comment]
@gep13  

I removed machines because with AU you don't need to have one, the example provided uses Appveyor as an example.

File: AutomaticPackages.md [hunk 1]
```diff
@@ -1,8 +1,17 @@
 # Automatic Packaging for Maintenance
-Automatic packaging is a process that **package maintainers can run on *their own machines* to keep the packages they maintain up to date**. It is not a required step for maintaining packages on the community feed (https://chocolatey.org/packages), but it is recommended you find a way to automate the delivery of packages to the community feed when there are updates if you are going to maintain more than 5 packages and you are not the software vendor for the packages you maintain.
+Automatic packaging is a process that **package maintainers can run on *their own* to keep the packages they maintain up to date**. It is not a required step for maintaining packages on the community feed (https://chocolatey.org/packages), but it is recommended you find a way to automate the delivery of packages to the community feed when there are updates if you are going to maintain more than 5 packages and you are not the software vendor for the packages you maintain.
```
Comment (ferventcoder) [inline AutomaticPackages.md]: Let's not remove the word machines. Or let's come up with something that can be used here.

Comment (ferventcoder) [inline AutomaticPackages.md]: Something to the effect of, these are available, but you are required to provide either of these solutions yourself.

**Comment by majkinetor** [comment]
IMO the current phrase is enough. It is a process you run on your own. There is nothing more to it. You dont need to *own* a machine to do that,.

**Comment by ferventcoder** [comment]
Fair enough

**Comment by ferventcoder** [comment]
My need is to ensure that folks understand this is not done by the community servers. This is why that sentence was included in the first place. I want to ensure that the removal of that word still conveys that message.

**Comment by ferventcoder** [comment]
It does appear to still convey the same message so 👍

**Pull request merged**